internal/infrastructure/kubernetes/infra.go

```diff
@@ -67,6 +70,7 @@ func NewInfra(cli client.Client, cfg *config.Server) *Infra {
 		DNSDomain:    cfg.DNSDomain,
 		EnvoyGateway: cfg.EnvoyGateway,
 		Client:       New(cli),
+		logger:       cfg.Logger.WithName("infra-kubernetes"),
```

Comment:
won't be have `infrastructure` twice https://github.com/envoyproxy/gateway/blob/55fc01fecf2819af479aa1a3e5f4af37b737a947/api/v1alpha1/envoygateway_types.go#L148 ? should this be `kubernetes` instead ?

Reply:
fixed